QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -15,6 +15,12 @@
     public static EventInput octocanumShiftingButton;
     public static EventInput recalibrateButton;
     public static EventInput strafingButton;
+    public static FloatInput forwardTrigger;
+    public static FloatInput backwardTrigger;
```

Comment:
To follow our current convention, it would probably be better to make these statuses and send the joystick values to them. This means we won't have to worry about null pointers.